src/main/java/Controller/Controller.java

```diff
@@ -1,33 +1,35 @@
 package Controller;
 
 import Contract.Contract;
+import Controller.Engine.Stockfish;
 import Model.Game;
 import Model.Pieces.PieceType;
 import Model.Position;
 import Model.Pieces.Piece;
-import View.MainView;
-import View.PositionView;
+import View.MainStage;
+import View.Board.PositionView;
+import javafx.application.Platform;
 
 import java.util.ArrayList;
 
 import static java.util.Objects.nonNull;
 
 public class Controller implements Contract.Listener, Contract.Observer {
 
-    Game game;
-    MainView mainview;
-    Piece clickedPiece = null;
+    private Game game;
+    private MainStage mainview;
+    private Piece clickedPiece = null;
+    private Boolean computerIsWhite = null;
```

Comment:
this isn't necessary, if it's not initialised then it is `null`. Both `game` and `mainview` above are also `null`.